make-options.json

```diff
@@ -15,11 +15,11 @@
         "AzureMysqlDeployment",
         "AzureMonitor",		
         "AzureNLBManagement",
-        "AzurePowerShell",
+        "AzurePowerShellV3",
```

Comment:
@stephenmichaelf  is this mismatched in releases/m134 too? does this file need to be cherry picked back?